### DISCUSSION TITLE
Support macOS Sequoia

### DISCUSSION THREAD

**Discussion by chris1111** [chris1111/D-LinkUtility-Package#30] [Show and tell]
This Utility now support macOS Sequoia 15

![Screenshot](https://github.com/user-attachments/assets/d67f8ad7-026c-4447-a6b6-1a41f802b770)

**Comment by rtuthaya** [comment]
Does this work with M1 ?

**Comment by rtuthaya** [comment]
I got his error - The install failed. (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance. An error occurred while running scripts from the package “OCD-LinkUtility.pkg”.) (1)

​Verifying SIP...DETAILS:HIDE
Install Progres.
Install Package
login item UNKNOWN
Install OCD-LinkUtility.pkg
0:126: execution error: installer: Package name is OCD-LinkUtility
installer: Installing at base path /
installer:PHASE:Preparing for installation…
installer:PHASE:Preparing the disk…
installer:PHASE:Preparing OCD-LinkUtility…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Configuring the installation…
installer:STATUS:
installer:%31.463346
installer:PHASE:Running package scripts…
installer:%33.356709
installer:PHASE:Running package scripts…
installer:%33.733884
installer:PHASE:Running package scripts…
installer:PHASE:Running package scripts…
installer:PHASE:Running package scripts…
installer:PHASE:Validating packages…
installer:%97.750000
installer: The install failed. (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance. An error occurred while running scripts from the package “OCD-LinkUtility.pkg”.) (1)
Install Utility-Package Done
Open PlistEdit
Install Done

**Comment by chris1111** [comment]
Provided the full Log in txt please, not only error
